gift_compassion/models/sponsorship_gift.py

```diff
@@ -187,11 +187,27 @@ def _compute_usd(self):
     @api.model
     def create(self, vals):
         """ Try to find existing gifts before creating a new one. """
+        gift_date = vals.get('gift_date')
+        if not gift_date:
+            invl = self.env['account.invoice.line']
+            dates = []
+            default = fields.Date.today()
+            for invl_write in vals.get('invoice_line_ids', [[3]]):
+                if invl_write[0] == 0:
```

Comment:
Why [[3]]?

Reply:
So that it goes in the else case when no values are set.